rollup.config.js

```diff
@@ -7,28 +8,25 @@ import json from 'rollup-plugin-json'
 import nodeResolve from 'rollup-plugin-node-resolve'
 import postcss from 'rollup-plugin-postcss'
 import pugPlugin from 'rollup-plugin-pug'
+import replace from 'rollup-plugin-replace'
 import { terser } from 'rollup-plugin-terser'
 import typescript from 'rollup-plugin-typescript'
 import url from 'rollup-plugin-url'
 import { dependencies } from './package.json'
 
 const external = [
   ...Object.keys(dependencies),
-  'crypto',
-  'events',
-  'fs',
-  'os',
-  'path',
-  'querystring',
-  'url',
-  'util',
   'chrome-launcher/dist/chrome-finder',
   'yargs/yargs',
 ]
 
-const plugins = [
+const plugins = (opts = {}) => [
   json({ preferConst: true }),
-  nodeResolve({ mainFields: ['module', 'jsnext:main', 'main'] }),
+  nodeResolve({
+    browser: !!opts.browser,
+    mainFields: ['module', 'jsnext:main', 'main'],
+  }),
+  replace({ 'process.env.NODE_ENV': JSON.stringify('production') }),
```

Comment:
The bundled `nanoid` package cannot work without replacing `process.env.NODE_ENV`.

ref: https://github.com/ai/nanoid/issues/127#issuecomment-529073587